src/LiveComponent/src/Normalizer/DoctrineObjectNormalizer.php

```diff
@@ -83,10 +84,29 @@ private function objectManagerFor(string $class): ?ObjectManager
         // todo cache/warmup an array of classes that are "doctrine objects"
         foreach ($this->managerRegistries as $registry) {
             if ($om = $registry->getManagerForClass($class)) {
-                return $om;
+                return self::ensureManagedObject($om, $class);
             }
         }
 
         return null;
     }
+
+    /**
+     * Ensure the $class is not embedded or a mapped superclass.
+     */
+    private static function ensureManagedObject(ObjectManager $om, string $class): ?ObjectManager
```

Comment:
See https://github.com/symfony/ux/issues/452#issuecomment-1242154695 for why this is needed.